docs/source/user-manual/navigation/tasks.rst

```diff
@@ -0,0 +1,28 @@
+Tasks
+=====
+
+All tasks can be found on the Tasks page. A task is created for each job that needs to be performed, such as running a Boefje,
+Normalizer or for generating a report. Each Boefje is performed on an object (such as a hostname, finding, DNS records, etc.) and each Normalizer on a raw file. Tasks have a status to show if the task is completed, scheduled, queued, etc. Each task contains meta and raw data that can be downloaded.
+This is the output, error message, proof, security hashes, etc. that belongs to the boefje or normalizer.
+Tasks can also be rescheduled and filtered to find specific tasks.
+
+Tasks are organised between the Boefje and Normalizer tabs. The image below shows what the Boefje tab could look like.
```

Comment:
This text and screenshots are outdated, the tasks page also shows reports tasks.

Can be updated in another PR.